Add macOS 11 onboarding guide (#3449)
actions/virtual-environments: README.md

| Windows Server 2019 | `windows-latest` or `windows-2019` | [windows-2019] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2019&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2019&redirect=1)
| Windows Server 2016 | `windows-2016` | [windows-2016] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2016&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2016&redirect=1)
```

> Ubuntu 16.04 is being deprecated. It is not recommended for new users. 

Reviewer: ```suggestion
Ubuntu 16.04 is deprecated and is not recommended for new users. 

```

> Ubuntu 16.04 is being deprecated. It is not recommended for new users. 
> If any of your workflows use Ubuntu 16.04, migrate to Ubuntu 20.04 or 18.04.

Reviewer: ```suggestion
Ubuntu 16.04 has been deprecated and will be removed on September 20, 2021.
Existing workflows using `Ubuntu 16.04` should migrate to `Ubuntu 20.04` or `Ubuntu 18.04`
```

```

```

> The macOS 11 virtual environment is currently provided as a private preview only.
> The "macos-latest" YAML workflow label still uses the macOS 10.15 virtual environment.
> ```
> > macOS 11 is now available for all existing Enterprises. Process of onboarding other plans is described on the [Onboarding page](./docs/macos-11-onboarding.md)

Reviewer: I will rather prefer to have a summary of the info on the first page instead of guiding users to another page. 
```suggestion
The macOS 11 virtual environment is currently in preview and available by default to existing customers on the Enterprise plan. 
New customers or customers on other plans should fill this form to request access to macOS 11 virtual environment. Please view our Big Sur guide for more details. 
The `macos-latest` YAML workflow label still uses the macOS 10.15 virtual environment.

```

Author: Thanks, updated
